packages/apps/dev-wallet/src/App/layout.tsx

```diff
@@ -31,14 +42,26 @@ export const Layout: FC = () => {
         }
       >
         <NavHeaderLinkList>
-          <Link to="/">
-            <Text bold>DX-Wallet</Text>
-          </Link>
-          <Link to="/networks">
-            <Text bold>Network</Text>
-          </Link>
+          <NavHeaderLink href="/">
```

Comment:
we need to use react-router links instead

Reply:
updated!